### PR TITLE
1956 UI prescription window

### PR DESCRIPTION
**Pull request by roxy-dao**
Part of 1956 (Won't build until #2038 is merged in... messed up order of PR 😭 )

# 👩🏻‍💻 What does this PR do? 
Translations and hooks for Prescription Window view, see in second PR

### CODE REVIEW

File: client/packages/invoices/src/Prescriptions/api/hooks/line/usePrescriptionLine.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    api.keys.detail(invoiceNumber),
+    () => api.get.byNumber(invoiceNumber),
+    { select }
+  );
+};
+
+export const usePrescriptionLine = (itemId?: string) => {
+  const selectItems = useCallback(
+    (invoice: PrescriptionRowFragment) => {
+      console.log('invoice: ', invoice);
```
Comment (mark-prins) [inline client/packages/invoices/src/Prescriptions/api/hooks/line/usePrescriptionLine.ts:24]: not required?

File: client/packages/invoices/src/Prescriptions/api/operations.graphql [hunk 2]
```diff
@@ -8,12 +8,118 @@ fragment PrescriptionRow on InvoiceNode {
...
+    totalCount
+  }
+
+  patient {
+    __typename
+    id
+    name
+    code
+    isDeceased
+    programEnrolments {
```
Comment (mark-prins) [inline client/packages/invoices/src/Prescriptions/api/operations.graphql:46]: are the program enrolments required for the prescription?

**Review by mark-prins — CHANGES_REQUESTED** [review]
woo! 🎉  exciting to see all of that come through.
I just have some import and translation changes please

File: client/packages/invoices/src/Prescriptions/api/operations.graphql (hunk 2 above)
Comment (roxy-dao) [reply client/packages/invoices/src/Prescriptions/api/operations.graphql:46]: The Patient Fragment that is being used for the PatientSearchInput has it, so if we remove it, the types don't match

File: client/packages/common/src/intl/locales/es/replenishment.json [hunk 3]
```diff
@@ -28,7 +28,6 @@
   "heading.target-quantity": "Cantidad objetivo",
   "heading.charges": "Cobros",
   "heading.stock-charges": "Cobros de inventario",
-  "heading.grand-total": "Gran total",
```
[the remark this answers is not in the document]
Comment (roxy-dao) [reply client/packages/common/src/intl/locales/es/replenishment.json:31]: [5bf3c10](https://github.com/openmsupply/open-msupply/pull/2038/commits/5bf3c10ad665b0ac0641113a9c36a9630a61048f)

**Review by mark-prins — APPROVED** [review]
👍  lovely

**Pull request merged**